### ISSUE TITLE
CSS in HTML body not working

### ISSUE THREAD

**Issue by noudadrichem** [dawidd6/action-send-mail#74]
When I add a simple HTML file with some CSS in the head it's not rendering CSS.

Expected: CSS to be rendered
Got: 0 read CSS

**Comment by dawidd6** [comment]
Any example? I don't know how it works tbh.

**Comment by sibr1011** [comment]
Hi [noudadrichem](https://github.com/noudadrichem) and anyone running into this problem,

I've noticed that, for example, _gmail_ often disregards css styles in the head. For this reason most html-templates using styles put them inline, such as `<div style="color: blue; "...></div>`. This can however be very time consuming. Therefore, I used the tool premailer to format my code automatically (there are many others out there). With a few tests and manual adjustments on top, I managed to get it to a level I was satisfied with. Hope it helps.

Btw dawid - maybe we could create a github action that runs before [action-send-mail](https://github.com/dawidd6/action-send-mail), which would take care of this problem. Just a thought.

Brian
